tests/test.spec.js

```diff
@@ -1,24 +1,24 @@
 import test from 'tape';
 
-test('#tape.spec', (t) => {
-  t.test('Test 1 should fail', (t) => {
+test('tape.spec', (t) => {
 
-    /*
-     * Run the tests and see that assertions can pass even
-     * if the entire test fails
-     */
+  t.test('Test 1 should fail', (t) => {
 
-    const actual = 5;
-    const expected = 5;
+    const actual = typeof Array.prototype.map;
+    const expected = 'object';
 
-    // t.pass(); This would generate a passing assertion
-    t.comment('Assertion passes');
-    t.equals(actual, expected);
-    // t.fail(); This would generate a failing assertion
-    t.comment('Assertion fails');
-    t.notEqual(actual, expected);
+    t.equals(actual, expected, 'typeof Array.prototype.map is "object"');
```

Comment:
It's more an example of testing in general than something that needs to be there.